react/utils/pixel.ts

```diff
@@ -27,27 +31,29 @@ export function handleItemClick(
   page: string,
   type: string
 ) {
-  return (term: string, position: number) => {
+  return (term: string, position: number, products: Item[]) => {
     push({
       page,
       event: EVENT_NAME,
       eventType: type,
       search: {
         term,
         position,
+        products
       },
     })
   }
 }
 
-export function handleSeeAllClick(push: (data: any) => void, page: string) {
+export function handleSeeAllClick(push: (data: any) => void, page: string, products: ISearchProduct[]) {
```

Comment:
I would just rename it to be more clear about which products are included here,  something like `suggestedProducts`, what do you think?
cause since the event is when you click the "see all" button, it can make someone else think that all the products of the search result should be here.

and I would suggest the same to the `handleAutocompleteSearch` function below